lms/routes.py

```diff
@@ -41,6 +41,10 @@ def includeme(config):
     config.add_route(
         "blackboard_api.courses.files.list", "/api/blackboard/courses/{course_id}/files"
     )
+    config.add_route(
+        "blackboard_api.courses.files.list_folder",
```

Comment:
"blackboard_api.courses.folder.files"? Not that important we only type this name once

Reply:
https://github.com/hypothesis/lms/pull/2950/commits/16d3d006991ee0a6eae3ee12210c3f656b9d089d